test/backend_fused_op.in.cpp

```diff
@@ -971,3 +971,315 @@ NGRAPH_TEST(${BACKEND_NAME}, split_var_len_parts)
 
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, lstm_cell_no_bias_no_peepholes)
+{
+    const size_t batch_size = 2;
+    const size_t input_size = 3;
+    const size_t hidden_size = 3;
+    const size_t gates_count = 4;
+
+    const auto X = make_shared<op::Parameter>(element::f32, Shape{batch_size, input_size});
+    const auto W =
+        make_shared<op::Parameter>(element::f32, Shape{gates_count * hidden_size, input_size});
+    const auto R =
+        make_shared<op::Parameter>(element::f32, Shape{gates_count * hidden_size, hidden_size});
+    const auto H_t = make_shared<op::Parameter>(element::f32, Shape{batch_size, hidden_size});
+    const auto C_t = make_shared<op::Parameter>(element::f32, Shape{batch_size, hidden_size});
+
+    const auto lstm_cell = make_shared<op::LSTMCell>(X, W, R, H_t, C_t, hidden_size);
+    auto function =
```

Comment:
Please don't call `decompose_op` in tests. Let's test the fused op, if the backend has direct support for it.